tendrl-api.spec

```diff
@@ -0,0 +1,90 @@
+Name: tendrl-api
+Version: 0.0.1
+Release: 1%{?dist}
+Summary: Collection of tendrl api extensions
+Group: Development/Languages
+License: LGPLv2+
+URL: https://github.com/Tendrl/tendrl-api
+Source0: %{name}-%{version}.tar.gz
+BuildArch: noarch
+
+BuildRequires: ruby
+BuildRequires: systemd-units
+
+Requires: ruby >= 2.0.0
+Requires: rubygem-i18n >= 0.7.0
+Requires: rubygem-json
+Requires: rubygem-minitest >= 5.9.1
+Requires: rubygem-thread_safe >= 0.3.5
+Requires: rubygem-mixlib-log >= 1.7.1
+Requires: rubygem-puma >= 3.6.0
+Requires: rubygem-rake >= 0.9.6
+Requires: rubygem-rack >= 1.6.4
+Requires: rubygem-tilt >= 1.4.1
+Requires: rubygem-bundler >= 1.13.6
+Requires: rubygem-tzinfo >= 1.2.2
+Requires: rubygem-etcd
```

Comment:
Any specific reason for not hardcoding the version dependency on rubygem-etcd like the others?